lib/bundles/inspec-compliance/http.rb

```diff
@@ -9,13 +9,21 @@ module Compliance
   # implements a simple http abstraction on top of Net::HTTP
   class HTTP
     # generic get requires
-    def self.get(url, token, insecure, basic_auth = false)
+    def self.get(url, token, insecure, user, basic_auth = false, automate = nil, server_type) # rubocop:disable Metrics/ParameterLists
```

Comment:
As discussed with @vjeffrey we think we can abstract that even further:

`def self.get(url, headers, insecure)`. This would allow us to pass in the required headers from outside:
```
headers = {
  'x-data-collector-token': token
}

headers = {
  'chef-delivery-user': user,
  'chef-delivery-token': token,
}

headers = {
  'Authorization', "Bearer #{token}",
}
```